security/secret-protection/docs/secret-protection.rst

```diff
@@ -0,0 +1,281 @@
+
+.. _secret-protection-tutorial:
+
+Tuturial: Secret Protection
+===========================
+
+Overview
+--------
+
+Confluent’s **Secret Protection** feature encrypts secrets in configuration
+files. Instead of storing passwords or other sensitive data as cleartext,
+**Secret Protection** encrypts the data within a configuration file itself.
+
+.. note::
+
+     You can either step through this tutorial, or run the `automated
+     demo <demo-secret-protection.sh>`__.
+
+
+Prerequisites
+~~~~~~~~~~~~~
+
+#. Download `Confluent Platform 5.4 <https://www.confluent.io/download/>`__ or greater
+
+#. Get the new `Confluent
```

Comment:
You can remove this step because CLI is back to being bundled with CP

Reply:
Updated